presto-product-tests/src/main/java/io/prestosql/tests/hive/TestAvroSchemaLiteral.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import org.testng.annotations.Test;
+
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tests.TestGroups.AVRO;
+import static io.prestosql.tests.utils.QueryExecutors.onHive;
+import static io.prestosql.tests.utils.QueryExecutors.onPresto;
+import static java.lang.String.format;
+
+public class TestAvroSchemaLiteral
+        extends HiveProductTest
+{
+    private static final String SCHEMA_LITERAL = "{\n" +
+            "  \"namespace\": \"io.prestosql.test\",\n" +
+            "  \"name\": \"product_tests_avro_table\",\n" +
+            "  \"type\": \"record\",\n" +
+            "  \"fields\": [\n" +
+            "    { \"name\":\"string_col\", \"type\":\"string\"},\n" +
+            "    { \"name\":\"int_col\", \"type\":\"int\" }\n" +
+            "]}";
+
+    @Test(groups = AVRO)
+    public void testHiveCreatedTable()
+    {
+        onHive().executeQuery("DROP TABLE IF EXISTS test_avro_schema_literal_hive");
```

Comment:
It would be better to limit `onHive()` usage for Hive specific operation because it's slow. Other places are also the same.

Reply:
This test is meant to test Hive created tables, so `onHive()` is used here.